code/mob/living.dm

```diff
@@ -1010,6 +1010,13 @@
 	var/obj/item/equipped_talk_thing = src.equipped()
 	if(equipped_talk_thing && equipped_talk_thing.flags & TALK_INTO_HAND && !message_mode)
 		equipped_talk_thing.talk_into(src, messages, secure_headset_mode, src.real_name, lang_id)
+
+	// AI phone fuckery
+	if(isAI(src))
+		var/mob/living/silicon/ai/ai = src
+		if(ai.phone.answered)
+			ai.phone.handset.talk_into(src, messages, secure_headset_mode, src.real_name, lang_id)
```

Comment:
Would rather this was only one `talk_into` call for clarity, use a variable to store the handset or put the logic for finding it into a proc